sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# SIG UI 2021 Annual Report
+
+**Authors:**
+
+Sebastian Florek ([@floreks](https://github.com/floreks))
+Marcin Maciaszczyk ([@maciaszczykm](https://github.com/maciaszczykm))
+Jeffrey Sica ([@jeefy](https://github.com/jeefy))
+
+- [Operational](#operational)
+- [Membership](#membership)
+- [Current initiatives and project health](#current-initiatives-and-project-health)
+
+## Operational
+
+**How are you doing with operational tasks in [sig-governance.md](/committee-steering/governance/sig-governance.md)?**
+
+The SIG primarily works asynchronously. In the past, our meetings would last less than five minutes and be relatively disruptive. Since the release of the 2.0 Dashboard, meeting attendance transitioned from optional to near-non-existent. Instead, we coordinate through the #sig-ui Slack channel and handle any project planning via GitHub. All chairs are regular contributors of the Dashboard subproject and help review PRs and triage issues. 
```

Comment:
we should think about a monthly at least - according to sig-governance y'all need to meet at least 3 times every week and have that opportunity for the community to reach you that way.

Reply:
ACK -- A monthly office hours type meeting would probably fill that gap. Thoughts?